src/text-field.jsx

```diff
@@ -55,6 +55,7 @@ const TextField = React.createClass({
     rowsMax: React.PropTypes.number,
     style: React.PropTypes.object,
     type: React.PropTypes.string,
+    underline: React.PropTypes.bool,
```

Comment:
Yeah that's true, I can see that being confused for a prop that accept a component.

Naming things is hard :smile_cat: